jsapp/js/constants.es6

```diff
@@ -347,6 +347,8 @@ new Set([
 
 export const NAME_MAX_LENGTH = 255;
 
+export const DVCOUNT_LIMIT_MINIMUM = 20;
```

Comment:
Please only add constants to this file that are used in more than one file. In this situation `DVCOUNT_LIMIT_MINIMUM` should be defined in `formLanding.es6`